CONTRIBUTING.md

```diff
@@ -453,4 +450,82 @@ New source code files should include the following header block:
 */
 ```
 
+## NEWS
+
+The purpose of the NEWS file is to record all the changes that are relevant for
+users and developers alike. These could be bug fixes, feature additions, syntax
+additions, or deprecations.
+
+### Format
+
+Each PHP version has a section, starting with the format:
+
+    {DD} {MMM} {YYY}, PHP {version}
+
+Such as:
+
+    06 Jun 2024, PHP 8.1.29
+
+In each section, entries are ordered by their extension. Extensions are listed
+alphabetically, with the exception of "Core", which MUST be listed first. Each
+extension's section starts with: `- {name}:\n`
+
+For each extension, entries are indented by two spaces, followed by `. ` (a
+period and a space).
+
+Entries MUST start with a capital and end with a period (`.`). The period goes
+outside of the `{issue-description}`.
+
+Each entry SHOULD be followed by the name of the contributor, or their GitHub
+handle if they so choose.
```

Comment:
FWIW: I've always used the lowercased php.net account name if one exists and the real name otherwise.